src/engine/filters/enginefiltermoogladder4.h

```diff
@@ -135,12 +136,10 @@ class EngineFilterMoogLadderBase : public EngineObjectConstIn {
 
             for (int i = 0; i < iBufferSize; i += 2) {
                 cross_mix += cross_inc;
-                m_postGain = m_postGainNew * cross_mix
-                        + startPostGain * (1.0 - cross_mix);
-                m_kacr = m_kacrNew * cross_mix
-                        + startKacr * (1.0 - cross_mix);
-                m_k2vg = m_k2vgNew * cross_mix
-                        + startK2vg * (1.0 - cross_mix);
+                m_postGain = static_cast<float>(m_postGainNew * cross_mix +
+                        startPostGain * (1.0 - cross_mix));
+                m_kacr = static_cast<float>(m_kacrNew * cross_mix + startKacr * (1.0 - cross_mix));
+                m_k2vg = static_cast<float>(m_k2vgNew * cross_mix + startK2vg * (1.0 - cross_mix));
```

Comment:
I am unsure here. I can remember that this code is sensitive for reduced precision so wee need to test it sorrowfully.

Reply:
This doesn't change anything. The members are already floats, so the typecast only makes this visible.

Reply:
This line is OK. I am concerned about everything. 
If the filter still sounds as before everything is OK.

Reply:
Sounds okay to me, but I'm on my laptop and the speaker don't sound good anyway :wink: Please double-check.

Reply:
I can confirm this.